packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -512,6 +513,35 @@ contract Colosseum_Initializer is L2OutputOracle_Initializer {
     }
 }
 
+contract SecurityCouncil_Initializer is Colosseum_Initializer {
+    address internal securityCouncilAddress;
+    uint immutable NUM_CONFIRMATIONS_REQUIRED = 2;
+    address[] owners = new address[](3);
+
+    // Test target
+    SecurityCouncil securityCouncilImpl;
+    SecurityCouncil securityCouncil;
+
+    function setUp() public virtual override {
+        Proxy proxy = new Proxy(multisig);
+        securityCouncil = SecurityCouncil(payable(address(proxy)));
+        securityCouncilAddress = address(securityCouncil);
+        super.setUp();
```

Comment:
Is there reason why `super.setUp()` is here?

Reply:
This may be needed later when testing calls through the Colosseum contract. There is no current usage. 
I'll remove the code and add it later if we need it.